lib/licensee/content_helper.rb

```diff
@@ -41,7 +41,7 @@ def length_delta(other)
     def similarity(other)
       overlap = (wordset & other.wordset).size
       total = wordset.size + other.wordset.size
-      100.0 * (overlap * 2.0 / total)
+      100.0 * ((overlap * 2.0 / total)**Math.log10(total))
```

Comment:
What does `**` do?

Reply:
Power operator. Base similarity to power of (log10 of wordset size). That's how dissimilarity is  scaled in this proposal.

I'm going to change total to wordset.size as the argument to log10 though, since there's no need for the scaling to be sensitive to the size of the candidate text, and it makes the scaling slightly less aggressive.